src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/ButtonBase.ButtonBaseAccessibleObjectTests.cs

```diff
@@ -235,6 +235,17 @@ public void ButtonBaseAccessibleObject_GetPropertyValue_ControlType_IsExpected_F
             Assert.False(buttonBase.IsHandleCreated);
         }
 
+        [WinFormsFact]
+        public void ButtonBaseAccessibleObject_GetPropertyValue_AutomationId_ReturnsExpected()
+        {
+            using var ownerControl = new SubButtonBase();
```

Comment:
The codebase was started before `new()` syntax was available, however now this is the preferred way when writing the new code:
```suggestion
            using SubButtonBase ownerControl = new();
```

Reply:
All suggestions changed according to the preferred code style.